### PR TITLE
Removes the ability to crawl or be thrown through plastic flaps.

### PR DESCRIPTION
**Pull request by lilacrhododendron**
:cl: 
balance: You can no longer crawl through plastic flaps, or be thrown through them. 
/:cl:

Being able to crawl at will made breaking into most departments extremely trivial, especially if you had maintenance access. Thanks to oranges for explaining it to me.

### CODE REVIEW

**Comment by zxaber** [comment]
Agreed. Some flaps (like for Delta's Robotics (and maybe the rest of Delta?)) are the sole barrier for entry, with no windoor in place. At this point, why even have access restrictions on the room's airlocks when flaps act as an open door policy?

**Comment by MMMiracles** [comment]
what if instead of doing this we fix the maps that have this issue

**Comment by Daxxed** [comment]
It was a fun way to sneak into places, or to get stuff off the cargo shuttle without having to hack and bolt 3 doors.

-biased miner main opinion

**Comment by Mickyan** [comment]
this will make it impossible for mobs (dead or alive) to ride the cargo conveyor belt into disposals fyi

you should also update the comment to reflect the change

**Comment by 81Denton** [comment]
If this prevents people from riding conveyor belts through plastic flaps, I'd rather tweak maps instead tbh

**Comment by Razharas** [comment]
This is retarded
Make it a force field then or a door of some kind, keeping it look like plastic flaps while having absolutely no characteristic one would expect from plastic flaps is confusing

**Comment by Potato-Masher** [comment]
Just tweak the maps if this is so much of an issue to avoid breaking other things like disposals.

Also note: 
>aquire screwdriver
>use on plastic flaps
>drag out plastic flaps from wall
>proceed with usual greytide

**Comment by Cyberboss** [comment]
Yeah, not like this

**Comment by MrDoomBringer** [comment]
subject_looks_like_bird_poop
@Cyberboss keep it closed

**Comment by optimumtact** [comment]
@MrDoomBringer are you offering to do the mapping? otherwise shut the cakehole and keep it shut

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
1/0 >> T/F the proc if not the whole file pls

File: code/game/objects/structures/plasticflaps.dm [hunk 1]
```diff
@@ -87,7 +87,7 @@
 			return 1
 		if(M.buckled && istype(M.buckled, /mob/living/simple_animal/bot/mulebot)) // mulebot passenger gets a free pass.
 			return 1
-		if((M.mobility_flags & MOBILITY_STAND) && !M.ventcrawler && M.mob_size != MOB_SIZE_TINY)	//If your not laying down, or a ventcrawler or a small creature, no pass.
+		if(!M.ventcrawler && M.mob_size != MOB_SIZE_TINY)	//If your not laying down, or a ventcrawler or a small creature, no pass.
```
Comment (ShizCalev) [inline code/game/objects/structures/plasticflaps.dm]: command is outdated.

**Comment by PKPenguin321** [comment]
Keep throwing through, remove crawling through

**Comment by Robustitron** [comment]
As an atmos main my fix for this was to put up a shocked grill behind the atmos flaps and occasionally check the flaps for dead assistants to take to cloning.

**Comment by zxaber** [comment]
Yeah, we're getting to the point where people are just blocking off flaps to keep tiders out, which begs the question of why even have flaps in the first place.

**Comment by Shadowflame909** [comment]
I would hope that you'd keep throwing people through these things. Only because my favorite thing to do is to table/disarm spam someone I wanna light-grief and toss them into a plastic flap/baby-jail

**Comment by kingoftheautism** [comment]
RIP fun rides in disposals

**Comment by Daxxed** [comment]
Replace bridge windoors by ones that can only be broken by robust weapons like lockers.
Add windoors to all regular department flaps, profit ???

**Comment by 81Denton** [comment]
@optimumtact I'm willing to do the mapping, I can get it done on the weekend probably.

**Comment by MrDoomBringer** [comment]
Wont this PR completely break MULEbots?

**Comment by Barhandar** [comment]
remove crawling instead

File: code/game/objects/structures/plasticflaps.dm [hunk 2]
```diff
@@ -71,24 +71,24 @@
...
-			return 0
+			return FALSE
 
 	if(ismecha(A))
-		return 0
+		return FALSE
 
 	else if(isliving(A)) // You Shall Not Pass!
 		var/mob/living/M = A
 		if(isbot(A)) //Bots understand the secrets
```
Comment (ShizCalev) [inline code/game/objects/structures/plasticflaps.dm]: @MrDoomBringer no.

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]
lol didn't go through the file cheeky

File: code/game/objects/structures/plasticflaps.dm (hunk 2 above)
Comment (MrDoomBringer) [reply code/game/objects/structures/plasticflaps.dm]: i c

**Review by Cyberboss — DISMISSED** [review]
Only crawling should be removed, resting/conveyor and throwing should still work. Flaps have been balanced around this forever

**Comment by anconfuzedrock** [comment]
First crawling takes away our resting
Today, crawling takes away our flaps
But what of tomorrow? what then? does the crawling take away your playing dead? Your vehicles? Your stamina? Your very lives? 
And there it is friends! the ugly truth. Crawling is merely becoming a slowdown with visuals!
Please make this work oranges I want to trust you but right now crawling does not seem promising

**Comment by subject217** [comment]
I mean, I'm willing to address the concern in this PR if there's a straightforward way to go about it that I can get to function. It's not an invalid concern, I agree that it'd be preferable to preserve the old behavior, especially moving through flaps on conveyor belts.

**Comment by BeeSting12** [comment]
I'm willing to do the mapping for it. It might take a while though.

**Comment by BeeSting12** [comment]
lol nvm denton did it already

**Pull request closed without merging**